Exercise_2_Currency_Ex/exchange.md

```diff
@@ -0,0 +1,40 @@
+# Purpose
+
+The purpose of this doc is to hold all the pseudocode for the `exchange.py` app. Future apps built using the exercism site will have a file similar to this one to help me create good habits of documentation and thinking about the problem before solutioning.
+
+Placing the pseudocode in this document will also allow me to keep the codebase clean, ensuring it only has comments for the specific functions called and steps taken.
+
+## Pseudocode
+
+`def estimate_value(budget, exchange_rate):`
+`def get_change(budget, exchanging_value):`
+`def get_value(denomination, number_of_bills):`
+`def get_number_of_bills(budget, denomination):`
+
+- No pseudocode written for these functions, please see the [README.md](/Exercise_2_Currency_Ex/README.md) for abbreviated instructions or the [code](exchange.py) for function details.
+
+`def exchangeable_value(budget, exchange_rate, spread, denomination):`
+
+``` Python
```

Comment:
code fences like this with a language shouldn't have spaces between the backticks an the language name, it may "work" but highly discouraged :)

Reply:
Updated in latest pull request, thanks!